### PR TITLE
[plg_twofactorauth_yubikey]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_twofactorauth_yubikey
### Testing Instructions

code review

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on becd6782c7afe6ca30da46677826f799a9e21a5c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10508">issues.joomla.org/joomla-cms/10508</a>.</sub>

File: plugins/twofactorauth/yubikey/yubikey.php [hunk 1]
```diff
@@ -120,7 +120,7 @@ public function onUserTwofactorShowConfiguration($otpConfig, $user_id = null)
 		}
 
 		// Is this a new TOTP setup? If so, we'll have to show the code validation field.
-		$new_totp = $otpConfig->method != $this->methodName;
+		$new_totp    = $otpConfig->method != $this->methodName;
```
Comment (andrepereiradasilva) [inline plugins/twofactorauth/yubikey/yubikey.php]: wht the space?

File: plugins/twofactorauth/yubikey/yubikey.php [hunk 2]
```diff
@@ -208,15 +208,15 @@ public function onUserTwofactorApplyConfiguration($method)
...
-		$yubikey = substr($data['securitycode'], 0, -32);
+		$yubikey      = substr($data['securitycode'], 0, -32);
 
 		// Check succeedeed; return an OTP configuration object
-		$otpConfig = (object) array(
+		$otpConfig    = (object) array(
 			'method'  => $this->methodName,
 			'config'  => array(
-				'yubikey'  => $yubikey
+			'yubikey' => $yubikey
```
Comment (andrepereiradasilva) [inline plugins/twofactorauth/yubikey/yubikey.php]: this i snot correct, in an array inside an array so, ident

File: plugins/twofactorauth/yubikey/yubikey.php (hunk 1 above)
Comment (wojsmol) [inline plugins/twofactorauth/yubikey/yubikey.php]: align with line 119

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 30fae29d02efc4d12595046bbe9c7ddca0c8a3ff<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10508">issues.joomla.org/joomla-cms/10508</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 30fae29d02efc4d12595046bbe9c7ddca0c8a3ff<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10508">issues.joomla.org/joomla-cms/10508</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10508">issues.joomla.org/joomla-cms/10508</a>.</sub>

**Pull request merged**